lib/rtsp/session/manager.ex

```diff
@@ -0,0 +1,120 @@
+defmodule Membrane.Protocol.RTSP.Session.Manager do
+  @moduledoc """
```

Comment:
@mat-hek Technically if someone wanted to use stateless Transport it could be used. The question is then, whether we want to allow it or encourage it. Having ready to go solution it is highly unlikely that scenario will happen.